bin/offline-dbg

```diff
@@ -133,6 +142,143 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # If no protocol listed, default to OpenFlow14 (enabled by default on ovs 2.8 on)
+    hex_ofp_version=$(cat ${CMD_DIR}/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo "OpenFlow14"
+    else
+        echo $version
+    fi
+}
+
+do_collect-sos() {
+    OPTIND=1
+    while getopts "h" opt; do
+        case ${opt} in
+            h)
+                usage
+                exit 0
+                ;;
+        esac
+    done
+    shift $(((OPTIND -1)))
+
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi
+    local SOS=$1
+
+    Dir_name=`tar -tf $SOS | head -1 | cut -f1 -d"/"`
```

Comment:
use "local dir_name"

Reply:
I was afraid this initial listing would grow linear to the size of the sos report. But I've run a couple of tests and it seems "head" makes it O(1) !! I'd say it closes the stdout fd once it has the first row and that makes tar exit
:)